Source/DafnyCore/Generic/ReporterExtensions.cs

```diff
@@ -11,7 +11,7 @@ public static class ErrorReporterExtensions {
     var usingSnippets = reporter.Options.Get(Snippets.ShowSnippets);
     var relatedInformation = new List<DafnyRelatedInformation>();
     foreach (var auxiliaryInformation in error.Aux) {
-      if (auxiliaryInformation.Category == RelatedMessageCategory) {
+      if (auxiliaryInformation.Category == RelatedMessageCategory || auxiliaryInformation.Category == AssertedExprCategory) {
```

Comment:
I'd rather not add edge-case code to a generic location such as `ReporterExtensions` to support a specific use-case (this PR). Ddi you consider using the existing `RelatedMessageCategory` ?